src/ragger/backend/speculos.py

```diff
@@ -177,27 +183,27 @@ def navigate_until_snap(self,
                             start_img_idx: int = 0,
                             last_img_idx: int = 0,
                             take_snaps: bool = True,
-                            timeout: int = 30,
+                            timeout: int = 5,
```

Comment:
5 seconds is too short a timeout if there are many screens in a flow, it will result in tests failing when they should succeed

Reply:
`timeout` could be a one-image parameter, which would be extended when another screenshot is encountered. That would bring more flexibility.

Reply:
Not sure what you mean :thinking:

Reply:
That makes very long `ragger` test, but ok. I'm switching back to 30sec.